types/HasteMap.js

```diff
@@ -72,4 +74,4 @@ export type HType = {|
   NATIVE_PLATFORM: 'native',
 |};
 
-export type HTypeValue = 0 | 1 | 2 | 3 | 'g';
+export type HTypeValue = $Values<HType>;
```

Comment:
this adds `'native'` as well. I guess that's OK?

Reply:
It's more correct, of course, but is it correct semantically? (I have no idea)

Reply:
No idea either, especially that HType has couple of keys with the same values, not sure what's the idea behind that 🤔

Reply:
Looking at the usage of `type: ?HTypeValue` in `ModuleMap.getModule` doesn't exactly make it less confusing :D It seems like the type is only used for `MODULE: 0, PACKAGE: 1`?
I think we need someone with more haste map expertise here

Reply:
@mjesun @rubennorte what do you think this type should be?